bundles/com.zeligsoft.domain.dds4ccm.ui/plugin.xml

```diff
@@ -566,6 +566,12 @@
             priority="Integer.MAX_VALUE"
             startKind="startup">
       </service>
+      <service
+            classname="com.zeligsoft.domain.dds4ccm.ui.services.CustomProfileDefinitionService"
+            id="org.eclipse.papyrus.uml.tools.service.IProfileDefinitionService"
+            priority="2"
```

Comment:
Out of curiosity: is priority 2 very low, or very high? I'm guessing low, which is consistent with this service contribution being a 'gap filler' for Papyrus Compare.